File: API/Models/MovieReservation.cs
using System.Collections.Generic;

namespace API.Models
{
    public class MovieReservation
    {
        public long MovieId { get; set; }
        public long SeatId { get; set; }
        public long UserId { get; set; }
        public bool WantParking { get; set; }
        public long StartDate { get; set; }
        public long EndDate { get; set; }

        public List<ParkingPlace> ParkingPlaces { get; set; }
    }
}

File: FE/src/app/_models/movieReservation.ts
import { User } from './user';
import { ParkingPlace } from './parkingPlace';

export class MovieReservation {
    movieId: Number;
    seatId: Number;
    userId: Number;
    startDate: Date;
    endDate: Date;
    wantParking: Boolean;
    parkingPlaces? : ParkingPlace[]; 
}